en/docs/security-announcements/cloud-security-bulletins/index.md

```diff
@@ -0,0 +1,7 @@
+---
+title: Cloud Security Bulletins
+summary: Overview of Cloud Security Bulletins
+category: security-announcements
+---
+
+# Cloud Security Bulletins
```

Comment:
Shall we also link from here to: 
- Choreo
- Asgardeo

and then inside choreo, asgardeo pages, have a index.md for each year. Like we have in advisories?